### ISSUE TITLE
reply to a bot's  previous message

### ISSUE THREAD

**Issue by kingofkillers91** [n3d1117/chatgpt-telegram-bot#169]
Is it possible to implement a feature where you can ask the bot something or you could ask the bot to do something just by replying to the bot's previous message?
Example
![Screenshot_20230403-000829~2](https://user-images.githubusercontent.com/48602339/229381701-e0edefe2-1e05-45ea-ba45-b8e20961a2e3.png)
In that conversation I wanted the bot to  translate into Italian that transcription he had made earlier. 
do you think you can do that? 
Great job btw. 
thank you

**Comment by n3d1117** [comment]
here's the easy way: just copy the transcript and append your prompt at the end of the message 😃

**Comment by kingofkillers91** [comment]
yeah, I could do that, but it is possible to automatize all this just by replying to the bot?

**Comment by n3d1117** [comment]
Not right now. Will keep this open in case someone wants to work on it!

**Comment by kingofkillers91** [comment]
no idea where to start with that, maybe @n3d1117 might provide a better answer
